CHANGELOG.md

```diff
@@ -11,13 +11,15 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 ### Adds
 
 - Adds the `pattern` prop to the `TextInput` component.
+- Adds the `isRenderedText` prop to the `HelperErrorText` component.
 
 ### Updates
 
 - Adds a third radio button to the `Header` component, separating search options between the Circulating Catalog, Research Catalog, and the NYPL.org website.
 - Temporarily removes the "log in" state from the `Header` component, the feature to read auth cookies, and the patron's name when logged in, and the log out link.
 - Truncate breadcrumb text if beyond 40 characters then add ellipsis at the end. If the breadcrumb text is truncated, the DS `Tooltip` component is used to display the full text in a tooltip when the text is hovered.
 - Updates the `SearchBar`'s `textInputProps` prop to include the following props to pass to the `TextInput` component: `isClearableCallback`, `max`, `maxLength`, `min`, and `pattern`.
+- Updates how the `CheckboxGroup`, `ComponentWrapper`, `DatePicker`, and `RadioGroup` components internally render the `HelperErrorText` component.
```

Comment:
Should we also include all form inputs that use `HelperErrorText`?  In order to use the latest a11y improvements for those other components, `v1.4.1` will be required.

Reply:
Yes, will update.